src/Plugins/Devices/DeviceProxy_LibUSB.cpp

```diff
@@ -454,10 +462,16 @@ void DeviceProxy_LibUSB::receive_data(uint8_t endpoint, uint8_t attributes, uint
 		break;
 	case USB_ENDPOINT_XFER_BULK:
 		*dataptr = (uint8_t *) malloc(maxPacketSize * 8);
-		rc = libusb_bulk_transfer(dev_handle, endpoint, *dataptr, maxPacketSize, length, timeout);
-		if (rc == LIBUSB_SUCCESS && debugLevel > 2)
-			cerr << "received bulk msg (" << *length << " bytes)" << endl;
-		break;
+		/* Edited by Skazza 20-10-2016 */
```

Comment:
`git blame` will show who wrote the code, these comments get out of date fairly quickly.